source/settings.json

```diff
@@ -273,10 +297,25 @@
       "default": "#0b639a"
     },
     {
-      "variable": "status_background_color",
-      "label": "Product status background color",
+      "variable": "badge_text_color_primary",
+      "label": "Badge text color (primary)",
+      "default": "#FFFFFF"
+    },
+    {
+      "variable": "badge_background_color_primary",
+      "label": "Badge background color (primary)",
       "default": "#C43F06"
     },
+    {
+      "variable": "badge_text_color_secondary",
+      "label": "Badge text color (secondary)",
+      "default": "#222222"
+    },
+    {
+      "variable": "badge_background_color_secondary",
+      "label": "Badge background color (secondary)",
```

Comment:
Can probably ditch the 'color' on these labels

Reply:
@nickendle are you sure? Other settings have the word color e.g. `Button background color` vs `Button background`

Reply:
@tmchow  yeah i don't think it really makes sense to have the word color repeated across every color setting. I didn't realize older settings still had it (we have started to move away from including it altogether with the introduction of the new color settings panel, it's likely this just hadn't been updated yet) -- I'm in favor of removing all instances of it unless you have strong feelings for keeping it. Only having it on some of them feels weird though

Reply:
@nickendle removed all instances of "color" from label